components/hab/install.ps1

```diff
@@ -208,8 +262,18 @@ Write-Host "Installing Habitat 'hab' program"
 $workdir = Get-WorkDir
 New-Item $workdir -ItemType Directory -Force | Out-Null
 try {
-    $Version = Get-Version $Version $Channel
-    $archive = Get-Archive $channel $version
+    # The $_patch may contain the /release string as well,
+    # This is fine because we only care about major/minor for this 
+    # comparison. 
+    $_major,$_minor,$_patch = $version -split ".",3,"SimpleMatch"
+
+    if( $Version -eq "" -Or $_major -ge 1 -Or $_minor -ge 89 ) { 
```

Comment:
if no `$version` is passed it will be `$null` and not empty. You could do:
```
if(!$Version -Or ([Version]::new($_major, $_minor) -ge [Version]::new("1.89")))
```

Reply:
I had to revert this change in order to be able to run on Windows 2008r2

Reply:
got it. but the revert seems off because it uses `$version` in all comparisons

Reply:
"revert" was the wrong word to use.  "Change back" would have been better, and then I copy/paste failed :face_with_head_bandage:

Reply:
oh on 2008R2 you would use:
```
$v1 = New-Object -TypeName Version -ArgumentList $_major,$_minor
$v2 = New-Object -TypeName Version -ArgumentList "0.89"
```

Reply:
^^^ works on all versions